CMakeLists.txt

```diff
@@ -116,8 +123,22 @@ if(SLEIGHHOME_DEFAULT)
 	endif()
 endif()
 
+if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
+	set(CMAKE_INSTALL_LIBDIR lib64 CACHE PATH "Output directory for libraries.")
```

Comment:
Why `lib64`? This seems too specific.

Reply:
I wasn't able to find a way to get it from CMake

Reply:
Is the `libdir="${prefix}/@CMAKE_INSTALL_LIBDIR@"` really necessary? What happens when you don't have `-L"${libdir}"` in there? Instead, you should probably set `PKGCONF_REQ_PUB` to `rz_core`.

Reply:
Good suggestion, it turned out unnecessary. Removed this completely.

Reply:
It is apparently necessary

Reply:
What error do you get when compiling something without it?

Reply:
@thestr4ng3r it's necessary for Windows where it's undefined.

Reply:
Ah I see, you also need it for installing. Then we should use `lib` instead of `lib64` as the fallback.

Reply:
Ok, switched to use `lib` instead.